# Define the goal
uint32 dobot_pick_id  # Specify which dishwasher we want to use
---
# Define the result
uint32 total_pick_up
---
# Define a feedback message
float32 percent_complete
